apisix/discovery/kubernetes/init.lua

```diff
@@ -571,6 +620,7 @@ local function multiple_mode_init(confs)
 
     for _, item in pairs(ctx) do
         start_fetch(item)
+        ngx.timer.every(1, update_token, item)
```

Comment:
running this timer every second doesn't sound like a good idea. Specially given that the token expires in X days.

Reply:
I moved it before calling `list_watch`.